src/components/booklist.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import AddBookForm from './addBookForm';
+
+const Booklist = () => {
+  // eslint-disable-next-line no-unused-vars
+  const [booklist, setBooklist] = useState([{
+    title: 'Boku no hero',
+    author: 'Kōhei Horikoshi',
+    id: 0,
+  }]);
+
+  return (
+    <section className="booklist-section">
+      {booklist.map((book) => (
+        <div key={book.id}>
+          <p>{book.title}</p>
+          <p>{book.author}</p>
+        </div>
+      ))}
+
+      <AddBookForm booklist={booklist} setBooklist={setBooklist} />
+    </section>
+  );
+};
+
+export default Booklist;
```

Comment:
- [ ] **Good job so far 👏 👏** However, It would be better to create a component called `Book` for displaying a single book and receiving the *title* and *author* as Props, 😀😎
- [ ] Each book should have a button for removing it (no functionality yet). and the created component should be reused to display a list of books. 👍